src/xrpld/app/tx/detail/VaultWithdraw.cpp

```diff
@@ -213,6 +207,8 @@ VaultWithdraw::doApply()
         return tecINSUFFICIENT_FUNDS;
     }
 
+    // These values are only going to decrease, and can't be less than 0, so
+    // there's no need for integer range enforcement.
```

Comment:
Leaving a comment here since the github doesn't let adding it to unchanged code. There is this code in `doApply()` just before the funds are transferred:

    auto const dstAcct = ctx_.tx[~sfDestination].value_or(account_);
    if (!vaultAsset.native() &&               //
        dstAcct != vaultAsset.getIssuer() &&  //
        dstAcct == account_)
    {
        if (auto const ter = addEmptyHolding(
                view(), account_, mPriorBalance, vaultAsset, j_);
            !isTesSuccess(ter) && ter != tecDUPLICATE)
            return ter;
    }

There is no need to check for native() and non issuer account - `addEmptyHolding()` has these checks.